randomisation/tests/test_utils.py

```diff
@@ -155,7 +155,7 @@ def test_stratification_balancing(self):
             totals[random_arm] += 1
 
         def check_arms_balanced(arms, diff, description):
-            values = [value for value in arms.values()]
+            values = list(arms.values())
```

Comment:
I think arms.values() might already return a list. Still better than my first attempt 🙂

Reply:
Tried removing the list() call, but got this error 

`'dict_values' object is not subscriptable`